mscore/resourceManager.ui

```diff
@@ -191,7 +191,7 @@
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
-          <number>1</number>
+          <number>0</number>
```

Comment:
This change may not be wanted?

Reply:
It's a mistake and I'll revert it later:)

Reply:
If that `currentIndex` is set (to 0) in the code, this here may be irrelevant. Seems it is not, but I guess it better would to make it imune vs. such accidental changes in the .ui file

Reply:
Qt Designer sets this field according to which tab is selected when I'm saving the ui file. And since I'm always editing the plugin related tabs and need to switch between them, sometimes it gets modified by mistake. I don't know if there's a better way to make the field immune.

But it only happens with default tab indices and probably has no more influence for now. There're two `currentIndex`s modified in the ui file, and I think it's easy to revert them when we squash commits or sometime else.